extensions/amazon-aws/src/components/dynamodb/query-form.tsx

```diff
@@ -15,10 +27,16 @@ interface QueryFormValues {
   pageLimit: string;
   useProjectionExpression: boolean;
   projectionExpression: string;
+  isDescending: boolean;
 }
 
 export const QueryForm = ({ table }: { table: Table }) => {
-  const { push } = useNavigation();
+  const [lastProjectionExpression, setLastProjectionExpression] = useCachedState<string>(
+    "last-projection-expression",
+    "",
+    { cacheNamespace: table.TableName },
```

Comment:
Could we add AWS account ID too?

Reply:
Oh yea good call regarding disambiguation, as two independent accounts might have the same table name. Although, the same table name might be intentional if cellular architecture is being used, for example, my-acct-beta and my-acct-prod might have the same table and can use the same projection for query. I think even in this case, it is better to let the use setup projection separately.

Reply:
Just using TableArn for cacheNamespace in the followup commit